### PR TITLE
Defines emsg id usage

### LINKED ISSUE #25: emsg id usage is not defined
The specification may want to give some recommendations with regard to using the id field in the emsg.

### PR DESCRIPTION
**Pull request by krasimirkolarov**
Resolves #25

### CODE REVIEW

**Comment by krasimirkolarov** [comment]
Resolves #25

File: av1-id3.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ aligned(8) class DASHEventMessageBox extends FullBox('emsg', version, flags = 0)
 
 `message_data` contains complete ID3 version 2.4 data \[[ID3]\].
 
+The id field is not restricted in this version of the specification.
```
Comment (cconcolato) [inline av1-id3.md]: minor editorial point. You can use quotes to ensure that the id field is styled as a syntactical element.

**Comment by krasimirkolarov** [comment]
Hi Cyril,

What is the protocol here, do you want me to do that edit and modify the spec, or is it done?

Krasimir

**Comment by cconcolato** [comment]
I would say the protocol is as follows:
As Editor feel free to update any of your pull request, or even create new ones, that are of editorial nature. 
If an issue is not editorial, everybody should open a new issue first (and potentially propose an associated pull request).

In this particular case, go ahead and update your pull request.

**Pull request merged**